test/fixtures/sim_pools.py

```diff
@@ -0,0 +1,79 @@
+from pandas import DataFrame
+import pytest
+
+from curvesim.pool.sim_interface import (
+    SimCurvePool,
+    SimCurveRaiPool,
+    SimCurveMetaPool,
+    SimCurveCryptoPool,
+)
+
+
+@pytest.fixture(scope="session")
+def sim_curve_pool():
+    return SimCurvePool(A=250, D=1000000 * 10**18, n=2, admin_fee=5 * 10**9)
+
+
+@pytest.fixture(scope="session")
+def sim_curve_meta_pool():
+    basepool = SimCurvePool(A=250, D=1000000 * 10**18, n=2, admin_fee=5 * 10**9)
+
+    def fixed_virtual_price():
+        return 10**18
+
+    basepool.get_virtual_price = fixed_virtual_price
```

Comment:
This is specific to the ParameterizedPoolIterator tests for metapools:
If D and D_base are both passed as parameters, setting the input D_base changes the basepool virtual price. Since metapool D depends on basepool virtual price, metapool.D() no longer equals the input D parameter. To avoid this, I just force a fixed virtual price.

Arguably we should change the parameter setters to avoid this gotcha.

For now, I moved this code into the ParameterizedPoolIterator tests, since other tests using these fixtures may not want this behavior.